docs/13-configuration.md

```diff
@@ -25,7 +25,16 @@ username: spark
 # docker_repo: <name of docker image repo (for more information, see https://github.com/Azure/aztk/blob/master/docs/12-docker-image.html)>
 docker_repo: aztk/base:spark2.2.0
 
-# custom_script: <path to custom script to run on each node> (optional)
+# Enable plugins
+plugins:
+  # - name: spark_ui_proxy
+  # - name: jupyterlab
+  # - name: jupyter
+  # - name: hdfs
+  # - name: rstudio_server
+
+# Allow master node to also be a worker(Default: true)
+# worker_on_master: true
```

Comment:
I think showing both possible values would be better: `worker_on_master: <true/false>`